backend/api/tests/test_admin_views.py

```
@@ -406,4 +406,130 @@ def test_promote_unauthorized(self, user_client, regular_user):
         
         response = user_client.post(url, data, format='json')
         
+        assert response.status_code == status.HTTP_403_FORBIDDEN
+
+@pytest.mark.django_db
+class TestWebcamRecordingsView:
+    """Test the WebcamRecordingsView admin API."""
+
+    @pytest.fixture
+    def test_webcam_recording(self, test_video, regular_user):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_webcam_recording`

(ANN201)